app/_src/mesh/features/rbac.md

```diff
@@ -1184,3 +1184,11 @@ conf:
 ```
 {% endnavtab %}
 {% endnavtabs %}
+
+## What should I do if I've locked myself out?
+
+If you remove the default `AccessRoleBinding` and `AccessRole`, you might find yourself locked out, unable to edit any resources. If you encounter this situation, you can regain access to the cluster by following these steps:
+
+1. Set the environment variable for the control-plane: `KUMA_ACCESS_TYPE: static`, and then restart the control-plane.
```

Comment:
Please use `configure the control plane` and put a link to https://docs.konghq.com/mesh/latest/documentation/configuration/#modifying-the-configuration